test/tests/message-format-test.sh

```diff
@@ -14,18 +14,26 @@ cp "$hook" ./.git/hooks/commit-msg
 git add . > /dev/null 2>&1
 
 # Assertions
-set -e
+# set -e
```

Comment:
I believe this needs to be uncommented. Although the assertions are great without propagating errors we need to know if a non-assertion statement fails in a test.